numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
Judicious ignorance is _exactly_ what PEP8 needs!

Rank 2+ arrays with variable precision / significant figures just can't be displayed in a way that makes PEP8 happy and is human-readable.